include/ProgramRepresentation/DisjointedPVAliasSets.h

```diff
@@ -0,0 +1,74 @@
+#ifndef DISJOINED_PV_ALIAS_SETS_H
+#define DISJOINED_PV_ALIAS_SETS_H
+
+#include "ProgramRepresentation/PVAliasSet.h"
+
+// manages a disjoint set (https://en.wikipedia.org/wiki/Disjoint-set_data_structure)
+// of program variable must-aliases.
+// each program variable is an element of no more than 1 set.
+class DisjointedPVAliasSets {
```

Comment:
Why "disjointed" instead of just "disjoint"? I don't think "disjointed" is a standard term.